Documentation/nvme-ocp-get-latency-monitor.txt

```diff
@@ -0,0 +1,55 @@
+nvme-ocp-get-latency-monitor(1)
+=========================================
+
+NAME
+----
+nvme-ocp-get-latency-monitor - Define and print get-latency-monitor value
+
+SYNOPSIS
+--------
+[verse]
+'nvme ocp get-latency-monitor' <device> [--sel=<select> | -S <select>] [--nsid=<nsid_val> | -n <nsid_val>]
+
+DESCRIPTION
+-----------
+Define get-latency-monitor.
+No argument prints current mode.
+
+The <device> parameter is mandatory and may be either the NVMe character
+device (ex: /dev/nvme0) or block device (ex: /dev/nvme0n1).
+
+This will only work on OCP compliant devices supporting this feature.
+Results for any other device are undefined.
+
+On success it returns 0, error code otherwise.
+
+OPTIONS
+-------
+
+-n <nsid>::
```

Comment:
The `--nsid` not described and also no description for the options `-n` and `--nsid`.